### ISSUE TITLE
Styling improvements

### ISSUE THREAD

**Issue by d-siganos** [realtimecoded/realtimeCoded#13]
At this point, the core of Padscape is more or less functional. Consequently, I think it is time I focused more on the look of the editor as well as the API. If you have any suggestions that you would like to see implemented soon, write them down here, and I will do them as soon as possible.

Notify stargazers @locness3 @KennethSweezy

**Comment by BonfireScratch** [comment]
@SimpleBinary @locness3 what is the next thing you would like to see implemented?

- Compiler support for other programming languages (just Python for now, and Java will possibly be next)
- Finally adding users and thus enabling authentication
- A landing page for Padscape with examples on how the use the editor
- A status website that shows when our servers are active (could be very useful for development purposes)
- A theme editor so that you can design and use a custom color theme in the editor

**Comment by locness3** [comment]
How would you handle Java in browser? For Pyhton, we can use Brython.
I would go with authentication for now.

**Comment by BonfireScratch** [comment]
@locness3 I was thinking of Brython as well. As for Java, we have JavaPoly.

**Comment by locness3** [comment]
Well, would be easier to implement Python/Brython first, as it seems like javaPoly needs participation from the backend to work.

**Comment by BonfireScratch** [comment]
@locness3 that's true. Originally I was.planning to use code from the TutorialsPoint editor, but it says that it is illegal to use the code in your own application. 😒

**Comment by locness3** [comment]
Well, if it says so in their Terms of use, which you have accepted by using the site, I guess it is.

**Comment by locness3** [comment]
Hmmm, why thumbs up?

**Comment by locness3** [comment]
Also, i think the subject's off-topic for this issue

**Comment by BonfireScratch** [comment]
It is, but this discussion will be deleted when SimpleBinary responds.

**Comment by BonfireScratch** [comment]
@locness3 
```html
<head>
    <script src="https://cdn.jsdelivr.net/npm/brython@3.8.8/brython.min.js"></script>
</head>
<body>
    <script type="text/python">
    print("hi")
    </script>

    <script>
    const log = console.log.bind(console);
    
    console.log = (...args) => {
        document.body.innerHTML = args;
    }

    brython();
    </script>
</body>
```

Yay!

**Comment by SimpleBinary** [comment]
@BonfireScratch I think you should focus on making sure the HTML/JS/CSS works very well before you start moving to other languages. I would probably just recommend making a landing page at this point.

**Comment by locness3** [comment]
Why all my comments were deleted??
